src/common/com/intellij/plugins/haxe/ide/module/HaxeModuleBuilder.java

```diff
@@ -51,6 +74,75 @@ public void moduleCreated(@NotNull Module module) {
     final CompilerModuleExtension model = (CompilerModuleExtension)CompilerModuleExtension.getInstance(module).getModifiableModel(true);
     model.setCompilerOutputPath(model.getCompilerOutputUrl());
     model.inheritCompilerOutputPath(false);
+
+    final Project project = module.getProject();
+    List<Pair<String, String>> sourcePaths = getSourcePaths();
+    String srcPath = null;
+
+    if ((sourcePaths.size() > 0)) {
+      srcPath = sourcePaths.get(0).getFirst();
```

Comment:
I *like* the idea of a QuickFix to add a main() method to a class.  I guess I don't really mind the ability to add a new class, either.  The trouble is that we're creating that class every time we create a new module.  And new modules aren't necessarily empty; they can be created from existing sources.  *Always* adding a new file is what really bothers me.  I think this functionality can exist, but NOT in the module create function.